build.gradle

```diff
@@ -33,45 +34,69 @@ allprojects {
 }
 
 subprojects {
+    apply plugin: 'com.github.ben-manes.versions'
     apply plugin: 'java'
     apply plugin: 'kotlin'
-    apply plugin: 'maven'
 
     sourceCompatibility = JavaVersion.VERSION_1_8
 
     repositories {
         maven {
-            url 'http://dl.bintray.com/kotlin/kotlin-eap-1.1'
+            url "https://plugins.gradle.org/m2/"
         }
         mavenCentral()
         jcenter()
     }
 
+    kotlin { experimental { coroutines 'enable' } }
+
+    dependencyUpdates.resolutionStrategy {
```

Comment:
Could you explain what this is doing?

Reply:
when i run `./gradlew dependencyUpdates` i don't want to see anything that is not marked as released